_posts/articles/2020-11-06-ten-tips-maintaining-long-term-docs-like-code.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: Ten tips for maintaining a long-term relationship with docs like code
+excerpt: "Learn about one team's journey to keep a long-term relationship with their docs-like-code system, TechDocs, which is open source and available to all."
+last_modified_at: Fri Nov  6 20:47:45 CST 2020
+categories: articles
+author: gary_niemen
+tags: [cicd, backstage, spotify, github, git, portal, internal, documentation, developer, docs]
+image:
+  path: /images/spotify/soundboard-han.ailes.jpg
+  caption: "[Flickr han.ailes](https://flic.kr/p/oVPTVm)"
```

Comment:
You can click https://flic.kr/p/oVPTVm to see the image... it reminded me of "backstage" and music which matches my great experience with Spotify, I'm a big fan of the service. :)